include/psa/crypto_extra.h

```diff
@@ -1901,6 +1901,331 @@ static inline struct psa_pake_operation_s psa_pake_operation_init( void )
     return( v );
 }
 
+/**
+ * \brief The context for PSA interruptable signing.
+ *
+ * \note  Contents not yet designed as implementation specific.
+ *
+ */
+struct psa_sign_interruptible_context_s
+{
+    unsigned num_ops;
+};
+
+typedef struct psa_sign_interruptible_context_s psa_sign_interruptible_context;
+
+/**
+ * \brief                       Set the maximum number of basic operations done
+ *                              in a row.
+ *
+ *                              If more operations are needed to complete a
+ *                              computation, #PSA_ERROR_IN_PROGRESS will be
+ *                              returned by the function performing the
+ *                              computation. It is then the caller's
+ *                              responsibility to either call again with the
+ *                              same parameters until it returns 0 or an error
+ *                              code; or to free the relevant context if the
+ *                              operation is to be aborted.
+ *
+ *                              It is strictly required that all input
+ *                              parameters and the restart context be the same
+ *                              on successive calls for the same operation,
+ *                              but output parameters need not be the same;
+ *                              they must not be used until the function
+ *                              finally returns 0.
+ *
+ *                              This only applies to functions whose
+ *                              documentation mentions they may return
+ *                              #PSA_ERROR_IN_PROGRESS.
+ *
+ * \param max_ops               Maximum number of basic operations done in
+ *                              a row. Lower (non-zero) values mean signature
+ *                              functions will block for a lesser maximum
+ *                              amount of time. The effect of changing this
+ *                              value will depend on the key used, the curve
+ *                              used and the underlying implementation, so we
+ *                              provide /c
+ *                              psa_sign_interruptible_get_num_ops() in order
+ *                              to help with tuning this value.
+ *
+ */
+void psa_sign_interruptible_set_max_ops( unsigned max_ops );
```

Comment:
It's unusual to see a naked "unsigned" type in the PSA headers. uintX_t seems to be the most common form.

Reply:
Changed this to size_t, as that seems to be the best fit here.
Also changed the char *s to uint8_t *s